PRIVACY.md

```diff
@@ -0,0 +1,48 @@
+
+For the purposes of this notice, WE refers to the entity of Flagrow and its proprietors, FORUM refers to a Flarum website that has installed Flagrow Bazaar and YOU refers to its owner, APPLICATION refers to Flagrow Bazaar, WEBSITE refers to flagrow.io. 
```

Comment:
I don't agree. I believe flagrow.io would be clearer here then WEBSITE. Because in the end flarum can also be seen as a website.